Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSource.java

```diff
@@ -212,6 +212,7 @@
         supportedSets.add("A25");
         supportedSets.add("DOM");
         supportedSets.add("BBD");
+        supportedSets.add("CM2");
```

Comment:
Don't add scryfall -- it haven't full cards list to download. All spoilers have only 20 cards, [not 400](https://magic.wizards.com/en/products/Commander-Anthology-2)